client/extensions/woocommerce/app/store-stats/index.js

```diff
@@ -32,14 +34,27 @@ class StoreStats extends Component {
 		};
 		return (
 			<Main className="store-stats woocommerce" wideLayout={ true }>
-				<Navigation unit={ unit } type="orders" />
+				<Navigation unit={ unit } type="orders" slug={ slug } />
 				<Chart
 					path={ path }
 					query={ ordersQuery }
 					selectedDate={ selectedDate }
 					siteId={ siteId }
 					unit={ unit }
 				/>
+				<StatsPeriodNavigation
+					date={ moment( selectedDate ) }
```

Comment:
might want to pass this as the string instead of a new moment instance. otherwise this has to re-render every time StoreStats renders.